catalog/transformer/catalog-transformer-geojson-input/src/main/java/ddf/catalog/transformer/input/geojson/GeoJsonInputTransformer.java

```diff
@@ -149,22 +150,8 @@ public Metacard transform(InputStream input, String id)
             }
         }
 
-        // find where the geometry goes
-        String geoAttributeName = null;
-        for (AttributeDescriptor ad : metacardType.getAttributeDescriptors()) {
-            if (AttributeFormat.GEOMETRY.equals(ad.getType()
-                    .getAttributeFormat())) {
-                geoAttributeName = ad.getName();
-            }
-        }
-
-        if (geoJsonGeometry != null) {
-            if (geoAttributeName != null) {
-                metacard.setAttribute(geoAttributeName, geoJsonGeometry.toWkt());
-            } else {
-                LOGGER.info("Loss of data, could not place geometry [{}] in metacard",
-                        geoJsonGeometry.toWkt());
-            }
+        if (geoJsonGeometry != null && StringUtils.isNotEmpty(geoJsonGeometry.toWkt())) {
+            metacard.setLocation(geoJsonGeometry.toWkt());
```

Comment:
We shouldn't inherently rely on how MetacardTypeImpl is implemented. It could change in the future to not include the Location attribute since it is an implementation.